lib/Mojolicious/Guides.pod

```diff
@@ -67,6 +67,12 @@ bytes or decoded characters in a Perl L<string|perlunifaq>, C<$bool> if the
 value just indicates true or false, C<$c> to denote a L<Mojolicious::Controller>
 object, or C<$app> to denote the L<application|Mojolicious> object.
 
+=item Signatures
+
+On Perl 5.20+ you can also use a C<-signatures> flag to enable support for
+L<subroutine signatures|perlsub/"Signatures">. The examples use signatures to
+show the parameters expected.
```

Comment:
I think this needs to be above variable names. And the second sentence can probably be rephrased a bit better,

Reply:
Moved and reworded